openassessment/staffgrader/models/submission_lock.py

```diff
@@ -91,6 +90,6 @@ def clear_submission_lock(cls, submission_uuid, user_id):
 
         # Only the owner can clear the lock
         if current_lock.owner_id != user_id:
-            raise SubmissionLockContestedError(_("Unable to clear submission lock"))
+            raise SubmissionLockContestedError("Unable to clear submission lock")
```

Comment:
Not translating?

Reply:
@muselesscreator , 📢 question about error-handling. I'm not sure where it makes sense to translate error messages: on the backend (so the frontend can just surface the message) or the frontend (so the backend message can be used for comparison/logic). Thoughts?

Edit: From discussion with @mattcarter, thinking it might make sense to provide both an error code (for comparison on the frontend) and pre-translated error message (for easy pass-through). Something like:

```
{ 'error_code': 'err_lock_contested', 'error_msg': (translatable)"Unable to clear submission lock" }
```